spec/unit/ssl/ssl_provider_spec.rb

```diff
@@ -505,28 +505,30 @@
       }.to raise_error(Puppet::Error, /The client certificate is missing from/)
     end
 
-    it 'loads the private key and client cert' do
-      ssl_context = subject.load_context
+    context 'loading private keys', unless: RUBY_PLATFORM == 'java' do
```

Comment:
What's the `unless: RUBY_PLATFORM == 'java'` for? I assume it's related to jruby and how the method we're using is broken there, but I'm just curious where/how `RUBY_PLATFORM` would get set.

Reply:
That's something that's set internally in Ruby, and changes between different OSes. When we are JRuby, we're running on the JVM, so the platform is listed as "java". We use it extensively throughout this repo to detect when we're on JRuby and skip things.

Reply:
gotcha, thanks!